demisto_sdk/commands/common/hook_validations/pack_unique_files.py

```diff
@@ -321,6 +325,7 @@ def validate_pack_meta_file(self):
             self._is_price_changed(),
             self._is_valid_support_type(),
             self.is_right_usage_of_usecase_tag(),
+            self.should_pack_not_be_hidden()
```

Comment:
i think changing it to `should_pack_be_hidden`, and replacing this with `not self.should_pack_be_hidden()` will be clearer.

Reply:
I agree, changed that.